CONTRIBUTING.md

```diff
@@ -47,47 +47,47 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+the extension out with. It is not an exhaustive showcase of DVC's features, so
+testers are encouraged to try other DVC repositories -- especially real-world
+cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Run `yarn setup:venv` from the
```

Comment:
[I] If you are going to go as far as saying install yarn you also need to mention run `yarn install` or `yarn setup:venv` will not work.

Reply:
We already state to install yarn on line 22

Reply:
Removing "Install yarn" to simplify.

> Personally I'd replace this whole bullet with a more generic instruction but don't want to deal with the implications rn. See https://github.com/iterative/vscode-dvc/pull/1928#discussion_r903058889 above